src/components/ConnectedTile.jsx

```diff
@@ -4,18 +4,16 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { translate } from 'cozy-ui/react/I18n'
 import { CONNECTION_STATUS } from '../lib/CollectStore'
-import { getAccountName } from '../lib/helpers'
 
-import { getConnectionStatusForTrigger } from '../ducks/connections'
+import {
+  getConnectionStatusForTrigger,
+  getKonnectorConnectedAccount
```

Comment:
This one totally my fault because I have still not cleaned the codebase `getKonnectorConnectedAccount` is deprecated and it is not returning the total amount of account (juste one if you're lucky) https://github.com/cozy/cozy-collect/blob/master/src/ducks/connections/index.js#L397